modules/siddhi-extensions/imageprocessing-objectdetection/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
WIll fix it. Fixed in https://github.com/hemikak/siddhi/commit/bb4059c56810d708c9e91cd7283b5891a5af8c60